scripts/etc-passwd

```diff
@@ -0,0 +1 @@
+pbnj:x:10001:10001::/home/pbnj:/usr/sbin/nologin
```

Comment:
newline

Reply:
just curious why? that I'm aware of, `/etc/passwd` files don't typically have a final newline.